src/tflite_cpu.cpp

```diff
@@ -0,0 +1,756 @@
+// Copyright 2023 Viam Inc.
```

Comment:
:laughing: I'm not convinced that adding this boilerplate into every file is useful, but at the very least I suspect the year is wrong here. I don't really care; leave as-is if you want. (TBH my first choice is to delete the whole 13-line comment entirely)

Reply:
This code was written back in 2023, and only lightly modified - but I can change the dates

Reply:
Including the short form license is recommended practice: https://apache.org/foundation/license-faq.html#Apply-My-Software.

Updating the year makes sense if this is a substantial revision, otherwise keeping the year is better.